apis/quarkus-common/src/test/java/io/stargate/sgv2/docsapi/api/common/token/impl/PrincipalTokenResolverTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright The Stargate Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.stargate.sgv2.docsapi.api.common.token.impl;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import io.quarkus.test.junit.QuarkusTest;
+import io.stargate.sgv2.docsapi.api.common.token.CassandraTokenResolver;
+import java.security.Principal;
+import java.util.Optional;
+import javax.enterprise.inject.Instance;
+import javax.inject.Inject;
+import javax.ws.rs.core.SecurityContext;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+@QuarkusTest
+class PrincipalTokenResolverTest {
+
+  @Inject // enabled by default
+  Instance<CassandraTokenResolver> tokenResolver;
+
+  SecurityContext securityContext;
+
+  @BeforeEach
+  public void setup() {
+    // TODO we should be able to do this directly on the field with @InjectMock, but for some reason
+    //  it doesn't work
+    securityContext = mock(SecurityContext.class);
```

Comment:
No idea why, everything looks the same as in sgv2-docsapi.

Reply:
Pretty sure connected to the fact no resource is defined in the `src/`.. Maybe a base profile in this module could add a simple example resource for all tests..

Reply:
Reverted in https://github.com/stargate/stargate/pull/1965/commits/49cd9c89ae71bd8349819a859ee6b07c45f39009, green now..